testing/ef_tests/check_all_files_accessed.py

```diff
@@ -27,20 +27,6 @@
     "tests/.*/.*/ssz_static/PowBlock/",
     # light_client
     "tests/.*/.*/light_client",
-    # LightClientStore
-    "tests/.*/.*/ssz_static/LightClientStore",
```

Comment:
I think we'll need to keep `LightClientStore` and `LightClientSnapshot` here as we don't support these types.